src/style.css

```
@@ -0,0 +1,104 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: normal;
+  font-style: normal;
+  src: url("../src/fonts/Avenir.ttc") format("opentype");
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  font-weight: normal;
+  font-style: normal;
+  src: url("../src/fonts/Avenir-Book.ttf") format("opentype");
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  font-weight: bold;
+  font-style: normal;
+  src: url("../src/fonts/Avenir-Heavy.ttf") format("opentype");
+}
```

Use numeric value for font-weight
```suggestion
@font-face {
  font-family: Avenir-Book;
  font-weight: 400;
  font-style: normal;
  src: url("../src/fonts/Avenir-Book.ttf") format("opentype");
}

@font-face {
  font-family: Avenir-Heavy;
  font-weight: 800;
  font-style: normal;
  src: url("../src/fonts/Avenir-Heavy.ttf") format("opentype");
}
```